Framework/DataHandling/src/LoadIsawDetCal.cpp

```diff
@@ -434,11 +432,7 @@ void LoadIsawDetCal::doRotation(V3D rX, V3D rY, DetectorInfo &detectorInfo,
 
   // Then find the corresponding relative position
 
-  // TODO: This needs checking. It would be expected that parent->getRot()
-  // should be all that is required to get the absolute rotation. This will
-  // affect WISH, but as there are no tests for this it is difficult to verify.
-  // The absolute rotation, when calling comp->getRotation, is taken as
-  // grandparentRot * parentRot * childrot
+  // TODO: This needs checking.
```

Comment:
Do you intend to leave this comment? Now it is totally unclear. Either remove it completely, or make clear what it is referring to.